packages/avatars/stories/examples/Default.tsx

```diff
@@ -0,0 +1,233 @@
+  badge: number;
+  surfaceColor: string;
+  backgroundColor: string;
+  foregroundColor: string;
+}
+
+export const Default: Story<IStoryProps> = ({
+  size,
+  status,
+  badge,
+  surfaceColor,
+  backgroundColor,
+  foregroundColor
+}) => {
+  const computedBadge = useMemo(() => {
+    if (badge === 0) {
+      return undefined;
+    }
+
+    if (badge > 9) {
+      return '9+';
+    }
+
+    return badge;
+  }, [badge]);
+
+  const computedStatus = useMemo(() => {
+    if (status === 'none') {
+      return undefined;
+    }
+
+    return status;
+  }, [status]);
+
+  return (
+    <Grid>
+      <Row>
+        <Col size="1">
+          <Row style={{ height: '10%' }}></Row>
+          <Row alignItems="center" style={{ height: '30%' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <LG>Icon</LG>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ height: '30%' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <LG>Image</LG>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ height: '30%' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <LG>Text</LG>
+            </Col>
+          </Row>
+        </Col>
+        <Col>
+          <Row style={{ marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <LG>Default</LG>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <LG>System</LG>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+              >
+                <UserIcon role="img" aria-label="Example SVG" />
+              </Avatar>
+            </Col>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+                backgroundColor={backgroundColor}
+                foregroundColor={foregroundColor}
+                isSystem
+              >
+                <ZendeskIcon role="img" aria-label="Zendesk" />
+              </Avatar>
+            </Col>
+          </Row>
+          <Row alignItems="center" style={{ backgroundColor: surfaceColor, marginBottom: '8px' }}>
+            <Col style={{ textAlign: 'center' }}>
+              <Avatar
+                size={size}
+                status={computedStatus}
+                badge={computedBadge}
+                surfaceColor={surfaceColor}
+              >
+                <img alt="" src={`images/avatar-3.png`} />
```

Comment:
Could src attribute be a string since it doesn't need string template features? I think there are a few other src attributes that also use a string template without using string template features.

Reply:
Correct. Prettier got aggressive here.